app/agency/utils.py

```diff
@@ -30,21 +36,66 @@ def update_agency_active_status(agency_ein, is_active):
             Agencies,
             agency_ein
         )
-        create_object(
-            Events(
-                request_id=None,
-                user_guid=current_user.guid,
-                auth_user_type=current_user.auth_user_type,
-                type_=AGENCY_ACTIVATED,
-                new_value={"ein": agency_ein},
-                timestamp=datetime.utcnow()
+        if is_active == "true":
```

Comment:
We should initialize a variable above (maybe after `is_valid_agency`) to be set to `eval_request_bool(is_active)` so that we can compare strictly boolean values.